packages/client/src/components/Layout.vue

```diff
@@ -31,7 +31,7 @@
         <b-nav-item to="/keywords" exact exact-active-class="active">Keywords</b-nav-item>
         <b-nav-item to="/users" exact exact-active-class="active" v-if="userRole === 'admin'">Users</b-nav-item>
         <b-nav-item to="/Agencies" exact exact-active-class="active">Agencies</b-nav-item>
-        <b-nav-item to="/Tenants" exact exact-active-class="active">Tenants</b-nav-item>
+        <b-nav-item to="/Tenants" exact exact-active-class="active" v-if="tenant.id === 1">Tenants</b-nav-item>
```

Comment:
I'll preface this with, I am not familiar how this all works together. 

Hard coding the tenant id in the front end seems bad. Is there another flag that can be added, "superuser" or the like to designate that the user can see across tenants? 

Is access to the tenant info being controlled here, or is this just whether to render the tab. Can you point me to where the tenant api determines whether a user can see other tenant info?

Reply:
There is no place (yet) in the tenant API that determines whether a user can see other tenant info. Currently it's the frontend that determines what users can see. The only users who should be able to see other tenants are tenants that belong to USDR (`tenant.id === 1`). 

I like the idea of adding a flag for readability and in case this functionality is extended in the future.

Reply:
yeah I would at minimum define a constant somewhere like `USDR_TENANT_ID` and reference that here. For server-side permissions, I suggested [over here](https://github.com/usdigitalresponse/usdr-gost/pull/189#discussion_r950583560) making a `requireUSDRAdminUser` middleware to enforce this for routes that read or write across tenant boundaries.

Reply:
Actually, I think it might be good for the display of this to also gate on `loggedInUser.role.name == 'admin'`.

Right now it looks like there are two routes in routes/tenants.js:
- `GET /api/organizations/:organizationId/tenants`: only ever returns a single tenant, the one you belong to; and then only if you are an admin. That should change to use e.g. `requireUSDRAdminUser` and then return all tenants.
- `PUT /api/organizations/:organizationId/tenants/:tenant`: right now has no permission check other than `requireAdminUser`; should use `requireUSDRAdminUser`.